File: encrypt.py
alpha = ('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV')
codes = ('!@#$%^&*()_+-?|\/}{][><_~=`1234567890')

alpha.split()
codes.split()

d = dict(zip(alpha,codes))


messages = input('enter message:')
def encoded(messages):
    code=''
    for i in range(len(messages)):
        if messages[i] in d.keys():
            code += d[messages[i]]
        else:
            code += messages[i]
    return code

def decoded(encry):
    code = ''
    for c in range(len(encry)):
        if encry[c] in d.values():
            keys = [i for i, r in d.items() if r==encry[c] ]
            code += ''.join(keys)
        else:
            code += encry[c]
    return code

encoded_message = encoded(messages)
print(encoded_message)
print(decoded(encoded_message))

File: encrypt.js
const p = ('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV');
const m = ('!@#$%^&*()_+-?|\/}{][><_~=`1234567890');

p.split('')
m.split('')

r = '';
const k = ('word in cell');
k.split('')
let dict ={};
for(i=0; i<p.length; i++){
    dict[p[i]] = m[i]
}

for(w of k) {
    if(w in dict){
        r += dict[w]
    }else{ r += w}
}
console.log(r)
